scripts/azure-linux_mac.yml

```diff
@@ -134,12 +140,26 @@ steps:
       docker ps -a
     fi
 
+    if [[ ( "$AGENT_OS" == "Linux" && "$TILEDB_AZURE" == "ON" ) ]]; then
+      # make sure docker is still running...
+      printenv
```

Comment:
I don't immediately need this, but we have a similar path for S3 so I'm just trying to keep it consistent. How about we put this in and I'll open a following PR to remove them both? I know that sounds a little backwards but it would at least isolate the discussion and we could consult Seth.

Reply:
I'm pretty sure it is cruft there too 😆 That's fine.